### ISSUE TITLE
Just to learn from you, have you ever reproduced the BiSeNet result reported in the paper without using auxiliary losses

### ISSUE THREAD

**Issue by CoinCheung** [GeorgeSeif/Semantic-Segmentation-Suite#155]
Sorry to bother you, but I went though the paper and yor code of the BiSeNet part, only to find that the paper claimed they used two auxiliary softmax cross entropy loss at layer2 and layer3 of the context path, which I did not found in your code. Have you reproduced the results in the paper with the code you pushed ?
